sample/perf/index.html

```diff
@@ -2,14 +2,11 @@
 <script type='module'>
   import Replicache, { REPMWASMInvoker, REPMHTTPInvoker } from './node_modules/replicache/out/mod.js';
 
-  const repmInvoker = location.hash == '#go' ?
-    new REPMHTTPInvoker('http://localhost:7002/') :
-    new REPMWASMInvoker();
-
-  console.log('Using repmInvoker', repmInvoker);
+  window.httpInvoker = new REPMHTTPInvoker('http://localhost:7002/');
```

Comment:
window.x :'(

Happy to fix it later but it does not really matter

The solution would have been to add the event listeners using code instead of html attributes.